src/components/Movies.vue

```diff
@@ -26,42 +34,67 @@ export default {
     data () {
         return {
             currentPage: null,
+            search: '',
+            go: false
```

Comment:
mislim da nema potrebe za ovim go
1. go kao ime je jako zbunjujuce, neko ko bi dosao da radi na projektu ne bi znao sta je morao bi da gleda kroz logiku za sta se koristi, tipa moviesLoaded bi bilo bolje
2. mislim da ce raditi i bez toga sve, samo namesti da ako nema movija da se renderuje prazna lista i to je to